db_cost_aws-bs.md

```diff
@@ -258,14 +262,17 @@ The `pwx-pg-cluster-x` is the postgres implementation using the volumes brought
 
 Connecting to the Postgres DB is as simple as pointing the app's `ORDER_DB_HOST` to `pwx-pg-cluster`. See section above
 
+Portworx can retain internal snapshots using a snapshot [schedule policy](https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/create-snapshots/scheduled/#creating-a-schedule-policy). This does not add to the cost as these snapshots are using the same storage pool.
```

Comment:
Added this note about snapshots since its mentioned below in the RDS config. To be consistent